shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -1226,6 +1226,9 @@ public void detachFromFlutterEngine() {
 
     if (flutterImageView != null) {
       flutterImageView.closeImageReader();
+      // Remove the {@link FlutterImageView} that was previously added by {@code
+      // convertToImageView}.
+      removeView(flutterImageView);
```

Comment:
Nice catch. I didn't think of a shared engine between multiple activities. Thanks for fixing!

Can we add a comment that mentions that this FlutterView is reused later, so any child view must be removed to avoid memory leaks?

Reply:
Done.